src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,114 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String addToDriversTableQuery = "INSERT INTO drivers (name, license_number) "
+                + "VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement addToDriversTableStatement

```

Comment:
Naming is good, but for simplicity you could also use simply:
```suggestion
                PreparedStatement statement

```
If you are working with one statement within one method.
**It is not an error, just my personal recommendation. Fix is optional**